### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Cifuuuu**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by Cifuuuu — COMMENTED** [review]
I think it i good

File: README.md [hunk 1]
```diff
@@ -14,32 +14,30 @@ _Collaborate and work together on GitHub._
...
-6. Enter a title for your pull request: `Update the game over message`.
-7. Enter a description for your pull request: `Update the game over message so people know how to play again!`
-8. Click **Create pull request**.
-9. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.
+1. On the pull request, click **Files changed**.
+1. Find the `index.html` changes.
+1. Hover your cursor next to the line numbers on the left side of the page.
+1. Click the blue plus icon.
+1. After the comment form appears, click the **Add a suggestion** button. <br>
+   ![add-a-suggestion-button](https://user-images.githubusercontent.com/97056108/184449714-61e8ee51-824a-48c1-9436-2dfd67f2c070.png)
```
Comment (Cifuuuu) [inline README.md:37]:
```suggestion
   ![add-a-suggestion-button](https://user-images.githubusercontent.com/97056108/184449714-61e8ee51-824a-48c1-9436-2dfd67f2c070.png)
```
```suggestion
   ![add-a-suggestion-button](https://user-images.githubusercontent.com/97056108/184449714-61e8ee51-824a-48c1-9436-2dfd67f2c070.png)
```
```suggestion
   ![add-a-suggestion-button](https://user-images.githubusercontent.com/97056108/184449714-61e8ee51-824a-48c1-9436-2dfd67f2c070.png)
```
```suggestion
   ![add-a-suggestion-button](https://user-images.githubusercontent.com/97056108/184449714-61e8ee51-824a-48c1-9436-2dfd67f2c070.png)
```
```suggestion
   ![add-a-suggestion-button](https://user-images.githubusercontent.com/97056108/184449714-61e8ee51-824a-48c1-9436-2dfd67f2c070.png)
```

File: .github/steps/-step.txt [hunk 2]
```diff
@@ -1 +1 @@
-1
+4
```
Comment (Cifuuuu) [inline .github/steps/-step.txt]:
```suggestion
i think it should be a 5
```

File: README.md [hunk 3]
```diff
@@ -14,32 +14,30 @@ _Collaborate and work together on GitHub._
...
+## Step 4: Suggest changes
 
-_Welcome to "Review pull requests"! :wave:_
+_Nice work reviewing that pull request :sparkles:_
 
-Let's get started by opening a pull request.
+Now that you have explored the different ways you can review a pull request it is time to learn how to use _suggest changes_.
 
-**What is a pull request?**: Collaboration happens on a pull request. The pull request shows the changes in your branch to other people. This pull request is going to keep the changes you just made on your branch and propose applying them to the `main` branch.
+**What is _suggest changes_?**: This feature enables you to recommend a change to a pull request that the author can commit with the push of a button.
```
Comment (Cifuuuu) [inline README.md]:
```suggestion
i think you are wrong
```

File: .github/steps/-step.txt [hunk 4]
```diff
@@ -1 +1 @@
-1
+5
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
i think it should be 555
```

File: .github/steps/-step.txt [hunk 5]
```diff
@@ -1 +1 @@
-1
+i think it should be 555
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
609123
```

File: .github/steps/-step.txt [hunk 6]
```diff
@@ -1 +1 @@
-1
+609123
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
609123ppp
```

File: .github/steps/-step.txt [hunk 7]
```diff
@@ -1 +1,2 @@
-1
+i want to change
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
make changes

```

**Review by Cifuuuu — COMMENTED** [review]
caddasdasd

**Review by incnotfun — COMMENTED** [review]
all good

File: .github/steps/-step.txt [hunk 8]
```diff
@@ -1 +1,3 @@
-1
+make changes
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
5
```

File: .github/steps/-step.txt [hunk 9]
```diff
@@ -1 +1,3 @@
-1
+5
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
77
```

**Pull request merged**